dynamic-plugins/wrappers/backstage-community-plugin-scaffolder-backend-module-servicenow-dynamic/dist-dynamic/package.json

```diff
@@ -10,9 +10,9 @@
   },
   "backstage": {
     "role": "backend-plugin-module",
-    "supported-versions": "1.32.4",
+    "supported-versions": "1.32.5",
     "pluginId": "scaffolder",
-    "pluginPackage": "@backstage/plugin-scaffolder-backend"
+    "pluginPackage": "@backstage-community/plugin-scaffolder-backend-module-servicenow"
```

Comment:
should this be more like in https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins/wrappers/backstage-plugin-scaffolder-backend-module-bitbucket-cloud-dynamic/package.json#L14-L15 ? 

You can run this command to clean up metadata...

```
./node_modules/@backstage/cli/bin/backstage-cli repo fix --publish
```

Reply:
Updated!